addons/sale_timesheet/models/project.py

```diff
@@ -39,6 +40,9 @@ def _default_timesheet_product_id(self):
         help="Employee/Sale Order Item Mapping:\n Defines to which sales order item an employee's timesheet entry will be linked."
         "By extension, it defines the rate at which an employee's time on the project is billed.")
     allow_billable = fields.Boolean("Billable", help="Invoice your time and material from tasks.")
+    billable_time = fields.Integer(
```

Comment:
Why an Integer ? 

Float is much more suited, and provide more range.

Reply:
Ooh my bad, it's a percentage. 

Then 
```suggestion
    billable_percentage = fields.Integer(
```